x/programs/cmd/simulator/src/util.rs

```diff
@@ -0,0 +1,16 @@
+use std::path::Path;
```

Comment:
not sure if this should be in a seperate `util` file

Reply:
This is only used in one place, you should just define it at the bottom of that file.